helpers/getFontsCollection.js

```diff
@@ -3,9 +3,8 @@
 const getFonts = require('./lib/fonts');
 
 const factory = globals => {
-    return function() {
-        const options = arguments.length === 1 ? arguments[arguments.length - 1] : null;
-        const fontDisplay = options && options.hash['font-display'] ? options.hash['font-display'] : null;
+    return function(options) {
+        const fontDisplay = options.hash['font-display'];
```

Comment:
Do we need to check for the presence of options here, or is options and options.hash always defined?

Reply:
If they aren't passing extra arguments then options and options.hash should be defined.  If nothing extra is passed options.hash is empty object.